src/long.ts

```diff
@@ -243,7 +254,15 @@ export class Long extends BSONValue {
    * @returns The corresponding Long value
    */
   static fromBigInt(value: bigint, unsigned?: boolean): Long {
-    return Long.fromString(value.toString(), unsigned);
+    // eslint-disable-next-line no-restricted-globals
+    Long.FROM_BIGINT_BIT_MASK ??= BigInt(0xffffffff);
```

Comment:
Remove caching

Reply:
bookkeeping: we tested this out in the huddle, and it doesn't affect performance and removing it will make the code less clunky